test/index_test.js

```diff
@@ -3,37 +3,55 @@
  * Module dependencies.
  */
 
-import { isValid, mask, type } from '../src';
-import * as isValidEin from 'is-valid-ein';
-import * as isValidItin from 'is-valid-itin';
-import * as isValidSsn from 'is-valid-ssn';
-import sinon from 'sinon';
+import * as einValidator from 'ein-validator';
+import * as itinValidator from 'itin-validator';
+import * as ssnValidator from 'ssn-validator';
 import should from 'should';
+import sinon from 'sinon';
+import { isValid, mask, sanitize, type } from '../src';
+
+/**
+ * Sandbox.
+ */
+
+const sandbox = sinon.sandbox.create();
 
 /**
  * Test.
  */
 
 describe('tin', () => {
-  const ein = '666234567';
-  const itin = '900700000';
-  const ssn = '031234567';
+  afterEach(() => {
+    sandbox.restore();
+  });
 
   describe('isValid()', () => {
     it('should return `false` is `tin` is invalid', () => {
       isValid('foobar').should.be.false();
     });
 
     it('should return `true` is `tin` is a valid `ein`', () => {
-      isValid(ein).should.be.true();
+      sandbox.stub(einValidator, 'isValid').returns(true);
```

Comment:
I think we should have at least one test using the dependencies, otherwise we may end up with errors only caught after release. /cc @fixe @nunofgs